### DISCUSSION TITLE
Should the "Contents" page be features in the PDF index?

### DISCUSSION THREAD

**Discussion by jmg-duarte** [joaomlourenco/novathesis#189] [Q&A] [answered]
I find myself navigating using the PDF's index, however there is no link to the "Contents" page.

Is this by design or a bug?

![image](https://user-images.githubusercontent.com/15343819/120246717-fa7fd100-c268-11eb-80dd-7c7b155e6ab0.png)

**Comment by joaomlourenco** [comment, marked as answer]
You are somewhat right.  You have to open the submenu "The Front Matter" to find the "Contents"… but then the "List of …" are at the top level.  I'll fix that by moving everything in.

<img width="946" alt="image" src="https://user-images.githubusercontent.com/2064643/120344313-6279fa00-c2f1-11eb-8b99-8aae0ab0e82b.png">

**Reply by joaomlourenco** [reply]
Solved in version 6.4.6 (just committed).

![image](https://user-images.githubusercontent.com/2064643/120400402-b73f6400-c335-11eb-82d7-0fb55ef5c2cb.png)
